calicoctl/commands/node/status.go

```diff
@@ -129,7 +136,19 @@ func drawTable(birdOut string) {
 
 		if s != "" {
 			col := []string{}
-			fields := strings.Fields(line)[3:6]
+			f := strings.Fields(line)[3:]
```

Comment:
so fields is of fixed size `fields := make([]string, 3)` anything more that that gets appended to the last one (e.g. in case of connection error) similar to python version
```go
// This is to include the extra info when the "Info" field is other than "Established"
// For example in case of "Mesh_fd80_24e2_f998_72d7__2 BGP  master   start  17:56:21 Active Socket: Connection closed"
// providing only "Active" in the Info section is not enough, so we append rest of the info into the last field
if len(f) > 3 {
				for _, e := range f[3:] {
					fields[2] = fields[2] + " " + e
				}
			}
```

So even if BIRD output changes, it won't break the code